webpush_channels/__init__.py

```diff
@@ -21,29 +21,29 @@
 
 
 DEFAULT_SETTINGS = {
-	'kinto.core.paginate_by': 100,
-	'flush_endpoint_enabled': False,
-	'retry_after_seconds': 3,
-	'cache_backend': 'kinto.core.cache.memory',
-	'permission_backend': 'kinto.core.permission.memory',
-	'storage_backend': 'kinto.core.storage.memory',
-	'project_docs': 'http://webpush-channels-broadcasting.readthedocs.io/',
-	'multiauth.authorization_policy': (
+    'kinto.core.paginate_by': 100,
+    'flush_endpoint_enabled': False,
+    'retry_after_seconds': 3,
+    'cache_backend': 'kinto.core.cache.memory',
+    'permission_backend': 'kinto.core.permission.memory',
+    'storage_backend': 'kinto.core.storage.memory',
+    'project_docs': 'http://webpush-channels-broadcasting.readthedocs.io/',
+    'multiauth.authorization_policy': (
 		'kinto.authorization.AuthorizationPolicy'),
-	'http_api_version': HTTP_API_VERSION
+    'http_api_version': HTTP_API_VERSION
 }
 
 
 def main(global_config, config=None, **settings):
-	if not config:
-		config = Configurator(settings=settings)
+    if not config:
+        config = Configurator(settings=settings)
 
-	config.add_settings({'kinto.project_name': 'webpush-channels'})
+    config.add_settings({'kinto.project_name': 'webpush-channels'})
 
-	kinto.core.initialize(config,
-                        version=__version__,
-                        default_settings=DEFAULT_SETTINGS)
+    kinto.core.initialize(config,
+			    version=__version__,
```

Comment:
Still a wrong indentation there, should be space and the config should be aligned with version and default_settings